definitions/grib2/products_crra.def

```diff
@@ -75,24 +75,73 @@ concept marsType(unknown) {
    }
 }
 
+# get the values from the 1st most outer loop which matter!
+#   aarays should be implemented in ecCodes as for bufr i.e. outerLoopLengthOfTimeRange[0] => #0#lengthOfTimeRange
+meta outerLoopLengthOfTimeRange element(lengthOfTimeRange, 0);
+meta outerLoopTypeOfStatisticalProcessing element(typeOfStatisticalProcessing, 0);
+meta outerLoopTypeOfTimeIncrement element(typeOfTimeIncrement, 0);
+
 # See GRIB-205 re no_copy
 # Cannot use typeOfProcessedData for stream. See GRIB-911
 concept marsStream(unknown) {
 
     oper = {
-      productDefinitionTemplateNumber = 8;
+      productDefinitionTemplateNumber = 0;
     }
     oper = {
-      productDefinitionTemplateNumber = 0;
+      productDefinitionTemplateNumber = 8;
     }
```

Comment:
is it just a swap but effectively doing the same thing?

Reply:
yes, not sure how it happened, just sort of beautification to keep the order